xclib/utf8.py

```diff
@@ -16,8 +16,11 @@ def unutf8(u, opts='strict'):
             traceback.print_exc()
             return 'illegal-utf8-sequence-' + dec
     else:
-        return u.decode('utf-8', opts)
-
+        try:
+            return u.decode('utf-8', opts)
+        except AttributeError:
+            pass
```

Comment:
shouldn't this be `return u` instead?

Reply:
Probably yes,
but I am not familiar with the code and I thought there could be cases where the object is not already decoded.
If that's not the case, then "return u" sounds like the most logical approach :-)